microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -96,33 +96,33 @@ Once you have completed the setup, return to this page and continue with the nex
 
 
 <!-- ====================================================================== -->
-## Step 2 - Set up Unity project for HoloLens 2 development
+## Step 2 - Set up Unity for HoloLens 2 development
 
-Before you can start developing a HoloLens 2 app with WebView2, you'll need to set up your Unity project for HoloLens 2 development. To prepare your project for mixed reality development, follow the steps in [Unity development for HoloLens 2](/windows/mixed-reality/develop/unity/unity-development-overview?tabs=arr,D365,hl2) in the Mixed Reality documentation. For a step-by-step walkthrough of configuring a Unity project, see [Set up a new OpenXR project with MRTK](/windows/mixed-reality/develop/unity/new-openxr-project-with-mrtk).
+Before you can start developing a HoloLens 2 app with WebView2, you will need to install Unity.  Follow the steps in [Choosing a Unity version and XR plugin](/windows/mixed-reality/develop/unity/choosing-unity-version) to complete installation of the toolset needed to build Mixed Reality applications.
 
 
 <!-- ====================================================================== -->
-## Step 3 - Install Mixed Reality Feature Tool and Mixed Reality Toolkit (MRTK)
+## Step 3 - Configure the Unity project for Mixed Reality development
```

Comment:
```suggestion
## Step 3 - Configure the Unity project for mixed reality development
```
If the meaning here is more "general industry tech" instead of "specifically Microsoft products"

Reply:
dup comment, tried to delete it.  GH PR having repaint/update issues.